src/frontend/test/features/featurefiles/sanity/login.feature

```diff
@@ -64,4 +61,15 @@ Feature: Log in to DPIA
       | DORGALE  | MPO     | Wrong1   |
       | DOND     | Drafter | Wrong2   |
       | ANDYA    | CPO     | Wrong3   |
+  
+  Scenario Outline: As <role> user, I am able to sign out successfully
+
+    Given I am on the login page
+    When I login with user <username> having role <role>
+    Then I sign out successfully
 
+    Examples:
+      | username | role    |
+      | DORGALE  | MPO     |
+      | DOND     | Drafter |
+      | ANDYA    | CPO     |
```

Comment:
Please add EOL - [an Empty blank line]

Reply:
Thanks a lot @kushal-arora-fw. Added new line to Login.feature and removed unnecessary spaces